web_switch_context_warning/static/src/js/switch_context_warning.js

```diff
@@ -0,0 +1,49 @@
+odoo.define("web_switch_context_warning.widget", function(require) {
+    "use strict";
+
+    var Widget = require("web.Widget");
+    var UserMenu = require("web.UserMenu");
+    var session = require("web.session");
+    // Show a big banner in the top of the page if the context has been
+    // changed in another tab or window (in the same browser)
+
+    if (!window.SharedWorker) {
+        // Not supported
+        return;
+    }
+    var SwitchCompanyWarningWidget = Widget.extend({
```

Comment:
In this new version maybe changing this to `SwitchContextWarningWidget` makes more sense. What do you think?

Reply:
Of course. now I'm going to change it.